lib/ansible/modules/windows/win_copy.py

```diff
@@ -44,59 +33,89 @@
     - Remote absolute path where the file should be copied to. If src is a
       directory, this must be a directory too.
```

Comment:
Will clarify further as copy does create the parent directories but unsure whether this is the expected behaviour.